pom.xml

```diff
@@ -93,7 +93,7 @@
     <metrics.client.version>0.4.3</metrics.client.version>
     <mockito.version>1.10.19</mockito.version>
     <slf4j.version>1.7.12</slf4j.version>
-    <vertx.version>2.1.6</vertx.version>
+    <vertx.version>3.2.1</vertx.version>
```

Comment:
Updated to specify the version you should use for Vertx 2.x, since there'll be no new releases of 0.4.z of this library I figured the specific release was adequate.